routing/road_graph.cpp

```diff
@@ -154,14 +166,40 @@ IRoadGraph::RoadInfo::RoadInfo(bool bidirectional, double speedKMPH, initializer
     : m_points(points), m_speedKMPH(speedKMPH), m_bidirectional(bidirectional)
 {}
 
-// IRoadGraph::CrossEdgesLoader ------------------------------------------------
-
-IRoadGraph::CrossEdgesLoader::CrossEdgesLoader(m2::PointD const & cross, TEdgeVector & outgoingEdges)
-    : m_cross(cross), m_outgoingEdges(outgoingEdges)
+// IRoadGraph::CrossOutgoingLoader ---------------------------------------------
+void IRoadGraph::CrossOutgoingLoader::LoadEdge(FeatureID const & featureId,
```

Comment:
ForEachEdge is implemented as void ForEachEdge(FeatureID const & id, RoadInfo const & info, TFn && fn)